### ISSUE TITLE
Simplify settings for Audible Accounts 

### ISSUE THREAD

**Issue by fusentasticus** [rmcrackan/Libation#862]
A new user may be confused by the format, especially when the table initially is presented as a single row. Specifically, there are buttons for rare actions (delete, and export) as the first fields in a row; this is a bit unconventional it would seem.  Please refer to (https://github.com/rmcrackan/Libation/issues/819#issuecomment-1984242711)

**Describe the solution you'd like**
I suggest to use right-click menu for these rare actions and to remove the two pseudo-fields.  Also, I suggest a light-grey rendering, italicized version of "Enter your account information" pre-filled into what will then be the first field. That's to make it clear to a first time user what to do! This message disappears when the box is clicked - that's a standard UI paradigm.

**Comment by Glaedr304** [comment]
I agree that the current login format isn't the most intuitive. An alternative solution I think would work well is having a button on the right of the table that opens the audible login window.

**Comment by wtanksleyjr** [comment]
Another item, unless this was already fixed: make sure the fields of the table have the same tab order as they do visual order ("tab order" refers to what field the cursor moves to when you press tab). When I filled in my accounts, the region field doesn't get tabbed to in the expected order; I think it might come after all of the others instead of where it visibly is. That has led people to not notice it's there.

**Comment by rmcrackan** [comment]
This is a fine request but after all this time, clearly I'm not actually going to get around to it. I think it's time for me to just admit defeat.
